lib/modules/dosomething/dosomething_signup/includes/SignupsController.php

```diff
@@ -2,6 +2,8 @@
 
 class SignupsController extends EntityAPIController {
 
+  protected $ignore_global_user = FALSE;
```

Comment:
I added an issue and [referenced it here](https://github.com/DoSomething/phoenix/pull/7018/files#diff-d06deaedbb4cf86687781d67ba7d5eddR118) since it's something I'm kind of concerned to update just for an experiment. Drupal doesn't necessarily instantiate these classes how we'd expect... it uses functions to call them via names saved as properties on the entityInfo stuff which makes it really difficult to find all occurrences of classes use and methods referenced. There seemed to be a very particular scenario as to why we have overridden the `save()` method on the controller and check against the global user when this module was created. Since we have no testing in place I'm rather concerned to mess with this too much, fearing I'll entirely break signups. This felt like a decent enough temporary solution for something we're not even sure will stick around. So if `/magicks` does end up working out well, we can take some time to really do some cleanup of the signups module.